### PR TITLE
added frogment editor

### PR DESCRIPTION
**Pull request by rajnandan1**
https://www.frogment.com/

Craft robust OpenAPI Specs Faster

A free openAPI spec editor and linter that breaks down your spec into fragments to make editing easier and more intuitive.

### CODE REVIEW

**Comment by daveshanley** [comment]
Is there a link to a github repo? a downloadable version?

**Comment by rajnandan1** [comment]
Will drop by discord. But really nice job with vacuum. 

There is no GitHub. It is nodejs web application

**Comment by daveshanley** [comment]
Ideally, a link to the source would be helpful to users, but not critical.

File: _data/tools.yml [hunk 1]
```diff
@@ -635,6 +636,16 @@
...
+  
+- name: Frogment Editor
+  category: gui-editors
+  language: Desktop / SaaS
+  link: https://www.frogment.com
+  description: |
+    A free openAPI spec editor and linter that breaks down your spec into fragments to make editing easier and more intuituve.
+  v2: false
+  v3: true
+  v3_1: true
```
Comment (daveshanley) [inline _data/tools.yml:649]: Please can you add an empty line; it will prevent merge conflicts downstream PRs

Comment (rajnandan1) [reply _data/tools.yml:648]: added

Comment (rajnandan1) [reply _data/tools.yml:648]: Hi.  Can you review the changes

**Pull request merged**